tap_github/schemas/commits.json

```diff
@@ -96,10 +96,176 @@
               "null",
               "string"
             ]
+          },
+          "contents_url": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "previous_filename": {
+            "type": [
+              "null",
+              "string"
+            ]
           }
         }
       }
     },
+    "author": {
+      "type": ["null", "object"],
+      "properties": {
+        "name": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "email": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "login": {
+          "type": ["null", "string"]
+        },
+        "id": {
+          "type": ["null", "integer"]
+        },
+        "node_id": {
+          "type": ["null", "string"]
+        },
+        "avatar_url": {
+          "type": ["null", "string"]
+        },
+        "gravatar_id": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "url": {
+          "type": ["null", "string"]
+        },
+        "html_url": {
+          "type": ["null", "string"]
+        },
+        "followers_url": {
+          "type": ["null", "string"]
+        },
+        "following_url": {
+          "type": ["null", "string"]
+        },
+        "gists_url": {
+          "type": ["null", "string"]
+        },
+        "starred_url": {
+          "type": ["null", "string"]
+        },
+        "subscriptions_url": {
+          "type": ["null", "string"]
+        },
+        "organizations_url": {
+          "type": ["null", "string"]
+        },
+        "repos_url": {
+          "type": ["null", "string"]
+        },
+        "events_url": {
+          "type": ["null", "string"]
+        },
+        "received_events_url": {
+          "type": ["null", "string"]
+        },
+        "type": {
+          "type": ["null", "string"]
+        },
+        "site_admin": {
+          "type": ["null", "boolean"]
+        },
+        "starred_at": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "committer": {
+      "type": ["null", "object"],
+        "properties": {
+          "name": {
+            "type": [
+              "null",
```

Comment:
Use the same indentation as other fields.

Reply:
Done

Reply:
I can still see indentation incosistency in other files. update it for each and every files.